src/v3/src/mocks/utils/utils.ts

```diff
@@ -66,7 +65,7 @@ export const getStubTransaction = (status: IdxStatus = IdxStatus.PENDING): IdxTr
   },
 });
 
-export const getStubTransactionWithNextStep = (): IdxTransactionWithNextStep => {
```

Comment:
Changing this type will impact many transformer tests because they all assume the NextStep will be present in the object (and by the flow, it will be). Any reason for removing this?

Reply:
hmm, still seeing it in my local branch, maybe place changed

Reply:
Oh I meant the type `IdxTransactionWithNextStep`, any reason you dont prefer the utility type which prevents the transformers from having to step around the optional `nextstep`?

Reply:
It's like another `Nullable`, `Nullish`